.sauce/config.yml

```diff
@@ -15,46 +15,10 @@ espresso:
   testApp: ./apps/mda-androidTest-1.0.10-12.apk
 suites:
   - name: "saucy barista - combined"
```

Comment:
The suite name never really made sense to me. Maybe a good opportunity to simplify it?

Reply:
sure!